src/common/colorspaces_inline_conversions.h

```diff
@@ -181,12 +210,12 @@ static inline float lab_f_inv(const float x)
 /** uses D50 white point. */
 static inline void dt_Lab_to_XYZ(const float *Lab, float *XYZ)
 {
-  const float d50[3] = { 0.9642, 1.0, 0.8249 };
+  const float d50[3] = { 0.96422f, 1.0f, 0.8249f };
```

Comment:
I did not spot that, there is a 2 added at the end of the first value!

Reply:
yes, it is a typo. Thanks @edgardoh !

Reply:
I have reverted this part now.

Reply:
Merci beaucoup pour ton aide Pascal !